internal/cli/performanceadvisor/slowoperationthreshold/slow_operation_threshold.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slowoperationthreshold
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "slowoperationthreshold"
```

Comment:
Just looking at the spec, I think we were planning camelCase here.
```suggestion
	const use = "slowOperationThreshold"
```